src/Calculator.cpp

```diff
@@ -8,4 +8,72 @@
 
 #include "Calculator.h"
 
-// Add Function definitions here.
\ No newline at end of file
+// Add Function definitions here.
+
+
```

Comment:
Add this to separate private and public methods inside of the C++ only
//*****************************************************************************/
//************** Calculator private methods ***************************/
//*****************************************************************************/